internal/gatewayapi/testdata/envoyproxy-tls-settings-invalid-ns.out.yaml

```diff
@@ -210,17 +210,17 @@ tcpRoutes:
   status:
     parents:
     - conditions:
+      - lastTransitionTime: null
+        message: Route is accepted
+        reason: Accepted
+        status: "True"
+        type: Accepted
```

Comment:
There is an inconsistency between the current Accepted status behavior of TCPRoute and HTTPRoute. The `Accepted` status should be `True`.

https://github.com/envoyproxy/gateway/blob/81e41c13520362f7b47b9c5e59af1b5bca279f14/internal/gatewayapi/testdata/envoyproxy-tls-settings-invalid-ns.out.yaml#L124-L136